src/components/Home.vue

```diff
@@ -1,13 +1,15 @@
 <script setup>
-import { defineProps } from "vue";
-import { ref, watchEffect } from "vue";
+import { ref, defineProps, watchEffect } from "vue";
+import { getLoggedInUser } from "..//stores/api.js";
 const props = defineProps({
   tasks: Array,
   isError: Boolean,
 });
 const currentTaskIndex = ref(0);
 const currentTask = ref(null);
 
+watchEffect(async () => await getLoggedInUser());
```

Comment:
Hmm. Why do we need to make this call in the home page, given we are making the same call in the Navbar?

Reply:
Maybe a better solution as @nbarnabee had suggested would be to call this method in App.vue and then pass a prop(loggedInUser) to other components the value will be a username or null if the user is not logged in

What I am struggling with is passing props through RouterView (<RouterView />)to views

Reply:
@Damilare1 any ideas?

Reply:
or pinia?

Reply:
hmm I think the challenge for me is that this method seems to be useful only in getting the username of the authenticated user. So I'm wondering why the method is called when the return value is unused.

Reply:
Also I think it's ok to import and call this function whenever we need the Authenticated user's username especially since it's a small application and we didn't have state management for this version road map.

Would be nice to have Pinia in the next cycle of development.

Reply:
I remember now, this is a leftover from edits that were made by @nbarnabee. It is okay to remove it. I will go ahead and remove it.